server/tests/bookings.test.js

```diff
@@ -281,7 +277,7 @@ describe('BOOKINGS TESTS', () => {
       .get('/api/v1/bookings')
       .set('Authorization', userToken)
       .end((err, res) => {
-        expect(res).to.have.status(200);
+        expect(res).to.have.status(404);
```

Comment:
This looks like your just forcing tests to pass but in actual sense, they don't. why did it change from a 200 to a 404?